File: backend/src/main/java/cohappy/backend/model/dto/CreateDebtDTO.java
package cohappy.backend.model.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class CreateDebtDTO {
    private String senderUserCode;
    private String receiverUserCode;
    private float amount;
    private String description;
}

File: frontend/app/src/main/java/cohappy/frontend/model/dto/request/CreateDebtDTO.kt
package cohappy.frontend.model.dto.request
import com.squareup.moshi.JsonClass

@JsonClass(generateAdapter = true)
data class CreateDebtDTO(
    val senderUserCode: String,
    val receiverUserCode: String,
    val amount: Float,
    var description: String? = null,
)